src/components/SectionScheduleCalendar/SectionScheduleCalendar.tsx

```diff
@@ -66,6 +68,14 @@ export function SectionScheduleCalendar({
     const [calendarDate, setCalendarDate] = useState(() => new Date());
     const [calendarView, setCalendarView] = useState<View>('work_week');
     const [interviewers, setInterviewer] = useState<User[] | undefined>(initialInterviewers);
+    const onChangeTime = ({ hours, minutes }: { hours: number; minutes: number }) => {
+        if (eventDetails) {
+            const newDate = toDate(eventDetails.originalDate.setHours(hours, minutes));
+
+            setEventDetails({ ...eventDetails, originalDate: newDate });
```

Comment:
You sure that it changes only exception, and not a whole event series?

Reply:
Yes, it doesn't touch the whole series of events, but creates a new event